src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' . $actual);
```

Comment:
@franzliedke I understand you. And I agree. But 
* it won't be any more readable when I convert the line added to this sandwich:
```return PHPUnit::assertEquals($code, $this->response->getStatusCode(), 'Expected status code '.$code . ', got '.$actual);```
* I hate interpolating the variables INTO the string the way you demonstrated: reminds me of spaghetti garbage codes I have hated my whole life. Of course this is matter of taste.

Overall, removing spaces around dots in the line added (forget the one above): @taylorotwell , do you really want me to squeeze it like that?!